src/Validators/Validator.Strings.cs

```diff
@@ -145,4 +145,22 @@ internal static void ThrowIfNotStartsWith(string value, string paramName, Except
             ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should start with '{str}'.");
         }
     }
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void ThrowIfContains(string value, string paramName, ExceptionCustomizations? exceptionCustomizations, string otherString, StringComparison comparisonType)
+    {
+        if (value.Contains(otherString, comparisonType))
+        {
+            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not contain '{otherString}'. (Comparison Type: {comparisonType})");
```

Comment:
Small change to align with the other exception messages (Here and below)
```suggestion
            ExceptionThrower.Throw(paramName, exceptionCustomizations, $"String should not contain '{otherString}' (Comparison Type: {comparisonType}).");
```